test/powershell/engine/Remoting/SSHRemotingAPI.Tests.ps1

```diff
@@ -2,20 +2,21 @@ Describe "SSH Remoting API Tests" -Tags "Feature" {
 
     Context "SSHConnectionInfo Class Tests" {
 
-        It "SSHConnectionInfo constructor should throw null argument exception for null UserName parameter" {
+        It "SSHConnectionInfo constructor should throw null argument exception for null UserName parameter" -Pending:$true {
 
+            # The test is pending because it is wrong or we have bug in code!
```

Comment:
Not sure.  But the test is incorrect (no longer needed) because a null UserName parameter is now allowed, and the test should be removed.